khl/util.py

```diff
@@ -2,7 +2,7 @@
 
 
 def unpack_id(obj):
-    """extract obj's id if not basic data type"""
+    """extract objet's id if not basic data type"""
```

Comment:
```suggestion
    """extract object's id if not basic data type"""
```

不过就用 obj 的话能对应参数名，方便理解

Reply:
`obj's` IDE 会划波浪线，不过不该也无所谓